system/ad_service_state_monitor/CMakeLists.txt

```diff
@@ -29,6 +29,11 @@ ament_auto_add_executable(ad_service_state_monitor
   ${AD_SERVICE_STATE_MONITOR_SRC}
 )
 
+# workaround to allow deprecated header to build on both galactic and rolling
+target_compile_options(ad_service_state_monitor PRIVATE
```

Comment:
Please apply the same solution as in https://github.com/autowarefoundation/autoware.universe/pull/720#discussion_r852040761

Reply:
@esteve addressed at https://github.com/autowarefoundation/autoware.universe/pull/716/commits/0fa477b3189323fd19c2b2f08542ffb398acee68.